src/home/org.freedesktop.home1.policy

```diff
@@ -10,8 +10,8 @@
         <vendor_url>http://www.freedesktop.org/wiki/Software/systemd</vendor_url>
 
         <action id="org.freedesktop.home1.create-home">
-                <description gettext-domain="systemd">Create a home</description>
-                <message gettext-domain="systemd">Authentication is required for creating a user's home.</message>
+                <description gettext-domain="systemd">Create a home area</description>
+                <message gettext-domain="systemd">Authentication is required to creat a user's home area.</message>
```

Comment:
Typo, “creat”, and similar all over the place. :)

Reply:
#14980.